web/portal/src/main/webapp/onboarding/jsp/reset_password.jsp

```diff
@@ -124,12 +124,6 @@
               <%}%>
 						</div>
 					</div>
-					<%} else if (success != null && !success.isEmpty()) {%>
```

Comment:
We should normally remove the returned errors and success from the OnboardingHandler: https://github.com/Meeds-io/gatein-portal/blob/develop/component/web/security/src/main/java/org/exoplatform/web/login/onboarding/OnboardingHandler.java#L151